.github/workflows/ci.yml

```diff
@@ -51,6 +51,9 @@ jobs:
       - name: Install Composer dependencies
         run: composer install
 
+      - name: Build
+        run: npm run build
```

Comment:
I have moved the build step to before testing since - if I remember correctly - I was getting errors and tests were failing. It looks like the PHP unit tests may rely on the build step but I may be wrong and I'll have another look tomorrow.

Reply:
I tested it locally and, as expected, we can lint early, before building the assets, however we cannot run PHP Unit tests at that time. It's because now we are using the `asset.php` files when enqueuing JS files. They are generated as part of the build process. When there is no asset file found, a [runtime exception is thrown](https://github.com/xwp/stream/pull/1541/files#diff-e6507b8ebd244ba19e7a955ee19c19f473c25ab5c82eb9162f960d76910bfd1fR360-R362) and tests fail.

In 79b2879 I moved the lint step before the build step.